### DISCUSSION TITLE
How to handle points that contain multiple points?

### DISCUSSION THREAD

**Discussion by madoleary** [tosdr/edit.tosdr.org#1019] [Ideas]
This keeps occurring in curation.

For example: https://edit.tosdr.org/points/1057

The google group discussion indicates that this is in fact numerous points. Do I separate it out into distinct points?

**Comment by michielbdejong** [comment]
yes. you can leave the first point as 1057, then create new points for the rest. write one comment on 1057 saying "point about bla split out to point X", and then on point X also leave a comment "split out from 1057". 

maybe we should write a short howto for backloggers? also explaining that it's important to click the 'source' link to the google group, and read the extra information that's often there.

**Comment by madoleary** [comment]
Yes, I've been leaving comments saying that I'm splitting it out into
multiple points. Also, agree about the how-to; there's usually good info in
the google group.
